src/NiceWrapper.cpp

```diff
@@ -227,6 +237,21 @@ void NiceWrapper::Stop() {
   }
 }
 
+void NiceWrapper::ParseRemoteSDP(std::string remote_sdp) {
+  string crfree_remote_sdp = remote_sdp;
+
+  // TODO: Improve this. This is needed because otherwise libnice will wrongly take the '\r' as part of ice-ufrag/password.

```

Comment:
I hate this bug; and in general it makes me a little nervous about how stable libnice actually is.